Doc/whatsnew/3.15.rst

```diff
@@ -70,6 +70,11 @@ Summary --- release highlights
 New features
 ============
 
+sqlite3
+-------
+
+* Support keyword completion in the :mod:`sqlite3` command-line interface.
+  (Contributed by Tan Long in :gh:`133393`.)
```

Comment:
Let's move this under the "Improved modules" section instead of "New features". The "New features" is reserved for PEP-sized items.